app/models/miq_server/server_smart_proxy.rb

```diff
@@ -32,7 +28,7 @@ def is_proxy_active?
   end
 
   def is_vix_disk?
-    self.has_active_role?(:SmartProxy) && capabilities && capabilities[:vixDisk]
+    self.has_active_role?(:SmartProxy) && has_vix_disk_lib
```

Comment:
Minor, but if this is a boolean column, you can use `has_vix_disk_lib?` which reads nicer.

Reply:
Also, minor, but since has_vix_disk_lib is a column, we should reverse this conditional, since checking a column is less expensive than checking for a role.